src/styles/blocks/days.scss

```diff
@@ -0,0 +1,27 @@
+.day {
```

Comment:
The class `.day` is used here, but in the HTML file, the class `calendar__day` is used. Ensure that the class names match between the HTML and SCSS files to apply the styles correctly.